### PR TITLE
Try to push main.js into npm

### PR DESCRIPTION
**Pull request by flakeed**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -131,4 +131,10 @@ dist
 
 *.js
 *.js.map
-*.d.ts
\ No newline at end of file
+*.d.ts
+
+# jest tests
+
+dist/tests/main.*
```
Comment (FreePhoenix888) [inline .gitignore]:
```suggestion
dist/tests
```

File: package.json [hunk 2]
```diff
@@ -11,7 +11,9 @@
     "url": "https://github.com/deep-foundation/react-use-are-packages-installed"
   },
   "scripts": {
-    "build": "tsc",
+    "build": "npm run build:lib && npm run build:test",
```
Comment (FreePhoenix888) [inline package.json]:
```suggestion
    "build-lib-and-test": "npm run build:lib && npm run build:test",
```

File: tsconfig.json [hunk 3]
```diff
@@ -19,6 +19,9 @@
     "resolveJsonModule": true,
     "esModuleInterop": true,
     "isolatedModules": true,
-    "declaration": true
-  }
+    "declaration": true,
+    "outDir": "dist"
+  },
+  "include": ["src/**/*.ts"],
+  "exclude": ["tests/**/*.ts"]
```
Comment (FreePhoenix888) [inline tsconfig.json:26]: Are tests building with this?

**Review by FreePhoenix888 — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 3 above)
Comment (flakeed) [reply tsconfig.json:26]: Yes

**Review by FreePhoenix888 — APPROVED** [review]

**Pull request merged**